tests/e2e/helpers/database.ts

```diff
@@ -143,6 +143,55 @@ export async function acceptLicenseTermsAndAddOSSClusterDatabase(databaseParamet
     await myRedisDatabasePage.clickOnDBByName(databaseName);
 }
 
+/**
+ * Accept License terms and add Sentinel database using api
+ * @param databaseParameters The database parameters
+ * @param databaseName The database name
+*/
+export async function acceptLicenseTermsAndAddSentinelDatabaseApi(databaseParameters: SentinelParameters, databaseName: string): Promise<void> {
+    await acceptLicenseTerms();
+    await discoverSentinelDatabaseApi(databaseParameters);
+    // Reload Page to see the database added through api
+    await t.eval(() => location.reload());
+    //Connect to DB
+    await myRedisDatabasePage.clickOnDBByName(databaseName);
+}
+
+/**
+ * Accept License terms and add RE Cluster database
+ * @param databaseParameters The database parameters
+ * @param databaseName The database name
+*/
+export async function acceptLicenseTermsAndAddREClusterDatabase(databaseParameters: AddNewDatabaseParameters, databaseName: string): Promise<void> {
```

Comment:
Same here, the `AddNewDatabaseParameters` is already include database name

Reply:
fixed thanks